eox_core/settings/common.py

```diff
@@ -20,9 +20,9 @@ def plugin_settings(settings):
     See: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/plugins/README.rst
     """
     settings.EOX_CORE_COMMENTS_SERVICE_USERS_BACKEND = "eox_core.edxapp_wrapper.backends.comments_service_users_j_v1"
-    settings.EOX_CORE_USERS_BACKEND = "eox_core.edxapp_wrapper.backends.users_j_v1"
+    settings.EOX_CORE_USERS_BACKEND = "eox_core.edxapp_wrapper.backends.users_l_v1"
```

Comment:
I think that we should not change the defaults here yet. At least we should wait for the official release of lilac

Reply:
But the DOD is plugins working on lilac. Do you think there will be a big change till the release?

Reply:
After these changes is the plugin working fine with Lilac? I think that there won'tbe a big change till the release, but in any I think that it is safer to keep the defaults without  Lilac. We are still using the main branch of eox-core in the SaaS and this could break that.

Reply:
Yes! The only issue I found is how to import the student module. Alright,  I'll follow your advice and let everyone know we are still using juniper backends as the default. Thank you!